fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java

```diff
@@ -3115,6 +3115,9 @@ public long loadDb(DataInputStream dis, long checksum) throws IOException, DdlEx
             connectContext.setDatabase(db.getFullName());
             Analyzer analyzer = new Analyzer(Env.getCurrentEnv(), connectContext);
```

Comment:
do not do this work in load db process.
should do this work in background thread. and add a sql command to show the process or error msg.